hook.go

```diff
@@ -536,15 +538,23 @@ func (h* Hook) updateAgentConfig(creds *credentials, installDir , buildPackLangu
 	h.Log.Debug("Successfully opened OneAgent config file %s for writing", agentConfigPath)
 	defer overwriteAgentConfigFile.Close()
 
+	// Add Info to ruxitagentproc.conf in case no values could be retrieved from the API
```

Comment:
This should be in the error handling section below line 446.
I think we should always get **something** back from the config API, so it makes sense to handle all of this in one place. So it's either "Config update failed" or it was successful.
But if it's possible to get an empty result from a successful GET request, this should also be handled above at the same place, but separatly. So we have that out of the way once we arrive at formatting the config properly.